src/libraries/System.Private.CoreLib/src/System/IO/Path.Unix.NoniOS.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO
+{
+    public static partial class Path
+    {
+        private static string DefaultTempPath = "/tmp/";
```

Comment:
granted it is private, does it matter if this is a field and other partial has property (i.e. same name, different assembly metadata)? was thinking maybe it should probably be:
```suggestion
        private static string DefaultTempPath => "/tmp/";
```